rustls/src/msgs/macros.rs

```diff
@@ -31,8 +31,8 @@ macro_rules! enum_builder {
             #[allow(dead_code)]
             $enum_vis fn as_str(&self) -> Option<&'static str> {
                 match self {
-                    $( $enum_name::$enum_var => Some(stringify!($enum_var))),*
-                    $(, $( $enum_name::$enum_var_nd => Some(stringify!($enum_var_nd))),* )?
+                    $( $enum_name::$enum_var_debug => Some(stringify!($enum_var_debug))),*
+                    $(, $( $enum_name::$enum_var_no_debug => Some(stringify!($enum_var_no_debug))),* )?
```

Comment:
I found the codecov warning message very interesting. When I tried the following mutation updates on these 2 lines `cargo test -p rustls --all-features` continued to succeed for me:

```suggestion
                    $( $enum_name::$enum_var_debug => None),*
                    $(, $( $enum_name::$enum_var_no_debug => panic!("XXX")),* )?
```

I would like to try contributing some test updates to improve the testing of these lines.